One-Mile-Up/base_cucumber_test_suite
# Add some tests

#### What does this PR do?
Adds 7 tests exercising the Cucumber/gherkin testing library with Rspec to test the functionality of the SauceLabs page

#### Where should the reviewer start?
At the first test/at the login screen

#### How should this be manually tested?
Repeat the actions described by the cucumber scenarios and make sure they execute the way the tests say they should

#### Any background context you want to provide?

#### Screenshots:
